app/helpers/dashboard/dashboard_helper.rb

```diff
@@ -22,5 +22,18 @@ def add_adviser_button(firm:)
         concat content_tag(:span, sr_label, class: 'visually-hidden')
       end
     end
+
+    def number_of_advisers(firm)
+      all_advisers_for_firm_and_trading_names(firm).size
+    end
+
+    def most_recently_edited_advisers(firm, limit = 3)
```

Comment:
I think keyword args are the way forward in this case.

```
def most_recently_edited_advisers(firm, limit: 3)
```


Much cleaner. (Aside, I've been using them a lot on the a different project and I was thinking of proposing "Keyword Arg Better Practice" as a Fireside discussion.)